src/SWIG_files/wrapper/TopoDS.i

```diff
@@ -1980,11 +1980,44 @@ Returns the type as a term of the shapeenum enum: vertex, edge, wire, face, ....
 ") ShapeType;
 		virtual TopAbs_ShapeEnum ShapeType();
 
+
+%extend{
+    bool __ne_wrapper__(const opencascade::handle<TopoDS_TShape> & other) {
+    if (self!=other) return true;
+    else return false;
+    }
+}
+%pythoncode {
+def __ne__(self, right):
+    try:
+        return self.__ne_wrapper__(right)
+    except:
+        return True
```

Comment:
I copied these method implementations from [TopoDS_Shape](https://github.com/JohannesVerherstraeten/pythonocc-core/blob/b220ed2ba4d5bebba98361a03bb4750dc8b9c7e6/src/SWIG_files/wrapper/TopoDS.i#L1620), but it's true that the try-except is a bit weird. 

Unless there is a specific reason to do it this way, I would remove the try-except from both the Shape and TShape implementations.

Reply:
Without the try except statement, something like:
```python
a_topods_shape != "Hi!"
```
would segfault because the right member is not a TopoDS_Shape. Using ``ìsinstance``` would certainly be better

Reply:
Ah yes I see. I'll make a new commit using isinstance